main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/aws-nuke/internal/awsnuke"
+	"github.com/gruntwork-io/gruntwork-cli/entrypoint"
+	"github.com/gruntwork-io/gruntwork-cli/logging"
+	"github.com/gruntwork-io/gruntwork-cli/shell"
+	"github.com/urfave/cli"
+)
+
+var VERSION string
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "aws-nuke"
+	app.Author = "Gruntwork <www.gruntwork.io>"
+	app.Version = VERSION
+
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "non-interactive",
+			Usage: "Assume yes for all interactive prompts",
+		},
+	}
+
+	log := logging.GetLogger("aws-nuke")
+	app.Action = func(c *cli.Context) error {
+		if c.NArg() < 1 {
+			return cli.NewExitError("insufficient number of arguments", 1)
+		}
+
+		id := c.Args().Get(0)
+
+		nuke := awsnuke.New(id)
+		ec2Instances, err := nuke.ListNonProtectedEC2Instances()
+		if err != nil {
+			log.Debugf("%v", err)
+		}
+
+		for _, instance := range ec2Instances {
+			log.Infof("ec2 instance: %s", instance.Id)
+		}
+		log.Infof("aws account: %s", id)
+		opts := shell.NewShellOptions()
+		opts.NonInteractive = c.Bool("non-interactive")
+		opts.Logger = log
+		yes, err := shell.PromptUserForYesNo("nuke resources?", opts)
```

Comment:
This prompt should be surrounded with tons of massive warnings about how this operation is destructive and cannot be undone.